osm2streets-java/StreetNetwork.java

```diff
@@ -0,0 +1,24 @@
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+class StreetNetwork {
+	static {
+		System.loadLibrary("osm2streets_java");
+	}
+
+	private static native long create(String osmXmlInput);
+
+	private static native String toGeojsonPlain(long pointer);
```

Comment:
I gave up fighting to make these normal methods on a `StreetNetwork` object. The example code just passes are raw `long` pointers: https://github.com/jni-rs/jni-rs/blob/f8e78e3583ce850c2699c5e72cb8517d8fff09a6/example/HelloWorld.java#L7.
I think we can do this properly with `jobject`, but I'll leave that for a followup